server/controllers/api_controller.go

```diff
@@ -92,7 +92,7 @@ func (a *APIController) Plan(w http.ResponseWriter, r *http.Request) {
 		a.apiReportError(w, http.StatusInternalServerError, err)
 		return
 	}
```

Comment:
Since you're merging to `main`, your fork's github actions build a new docker image of your code.

https://github.com/ghaiszaher/atlantis/pkgs/container/atlantis

Have you used this new image in your atlantis setup and seen success with this queuing functionality?

Reply:
I just tried it and it's working properly.

Reply:
Wonderful! Please add information regarding this end to end test under a `## tests` section in the PR body as that will help reduce concerns.